packages/medusa/src/api/routes/store/carts/create-cart.ts

```diff
@@ -241,6 +241,9 @@ export class StorePostCartReq {
   @IsOptional()
   context?: object
 
+  @IsOptional()
```

Comment:
todo: Add property to `@schema StorePostCartReq`

Reply:
Done